server/util/status/status.go

```diff
@@ -263,3 +263,12 @@ func MetricsLabel(err error) string {
 	// errors).
 	return status.Code(err).String()
 }
+
+// Retryable returns false if the error is a configuration error
+// that will persist, even despite retries.
+func Retryable(err error) bool {
```

Comment:
this logic is a bit specific to RBE, e.g. we probably wouldn't want to use this to decide whether a cache RPC should get retried - might make sense to put it in the `rexec` package